runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -90,6 +89,20 @@
 		final JMenuItem openXpTracker = new JMenuItem("Open online tracker");
 		openXpTracker.addActionListener(e -> LinkBrowser.browse(XpPanel.buildXpTrackerUrl(client.getLocalPlayer(), skill)));
 
+		// Create skill icon panel for pause icon overlay
+		SkillIconPanel skillIconPanel = new SkillIconPanel(iconManager.getSkillImage(skill));
+
+		// Create pause menu
+		pause = new JMenuItem("Pause");
+		pause.addActionListener(e ->
+		{
+			boolean paused = pause.getText().equalsIgnoreCase("Pause");
```

Comment:
Because these strings are used to store state they should be staticfinal constants